MAINTAINERS.yml

```diff
@@ -0,0 +1,1052 @@
+        - samples/drivers/watchdog/
+        - tests/drivers/watchdog/
+    labels:
+        - "area: Watchdog"
+
+"Drivers: WiFi":
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - tbursztyka
+    collaborators:
+        - jukkar
+    files:
+        - drivers/wifi/
+    labels:
+        - "area: Wifi"
+
+"Drivers: WiFi es-WiFi":
+    # Provisional entry, to be corrected
+    collaborators:
+        - loicpoulain
+    files:
+        - drivers/wifi/eswifi/
+    description: >-
+        Inventek es-WiFi
+
+"File Systems":
+    # Provisional entry, to be corrected
+    status: orphaned
+    collaborators:
+        - nashif
+        - nvlsianpu
+        - pabigot
+        - vanwinkeljan
+        - Laczen
+        - wentongwu
+    files:
+        - include/fs/
+        - include/fs.h
+        - samples/subsys/fs/
+        - subsys/fs/
+        - tests/subsys/fs/
+    labels:
+        - "area: File System"
+
+Kconfig:
+    status: maintained
+    maintainers:
+        - ulfalizer
+    files:
+        - scripts/kconfig/
+    files-regex:
+        - Kconfig[\w._-]*$
+    labels:
+        - "area: Kconfig"
+    description: >-
+        See https://docs.zephyrproject.org/latest/guides/kconfig/index.html and
+        https://docs.zephyrproject.org/latest/guides/porting/board_porting.html#default-board-configuration
+
+Kernel:
+    status: maintained
+    maintainers:
+        - andyross
+    collaborators:
+        - andrewboie
+    files:
+        - doc/reference/kernel/
+        - include/kernel*.h
+        - kernel/
+        - tests/kernel/
+    labels:
+        - "area: Kernel"
+
+Logging:
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - nordic-krch
+    files:
+        - include/logging/
+        - samples/subsys/logging/
+        - subsys/logging/
+        - tests/subsys/logging/
+    labels:
+        - "area: Logging"
+
+"Native POSIX and POSIX arch":
+    status: maintained
+    maintainers:
+        - aescolar
+    files:
+        - arch/posix/
+        - boards/posix/native_posix/
+        - drivers/*/*native_posix*
+        - drivers/*/*/*native_posix*
+        - dts/posix/
+        - include/arch/posix/
+        - scripts/valgrind.supp
+        - soc/posix/
+        - subsys/debug/asan.c
+        - tests/boards/native_posix/
+    labels:
+        - "area: native port"
+    description: >-
+        POSIX architecture and SOC, native_posix board, and related drivers
+
+Networking:
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - jukkar
+    collaborators:
+        - tbursztyka
+        - pfalcon
+    files:
+        - drivers/net/
+        - include/net/
+        - samples/net/
+        - subsys/net/
+    files-exclude:
+        - samples/net/sockets/coap_*/
+        - samples/net/lwm2m_client/
+        - subsys/net/lib/coap/
+        - subsys/net/lib/lwm2m/
+        - subsys/net/lib/openthread/
```

Comment:
Shall we add a separate entry for OpenThread then?

Reply:
CC @jukkar.
This I wrote due to the current rules in CODEOWNERS. But I certainly need help from you guys here.
Regarding your exact question: It is possible (Maybe you want to take a look at the BT entries as an example of another complex area). There is the option of splitting some part fully into its own area. Or also of adding a sub-area to an area, where that sub-area may have its own maintainer, or just extend with some extra collaborators.